packages/core/config/babel-preset.js

```diff
@@ -13,6 +16,21 @@ const TARGETS_NODE = '12.16.1'
 // instead of corejs: 3, since with corejs: 3 will not be injected modules which
 // were added in minor core-js releases.
 const CORE_JS_VERSION = '3.6'
+const DB_INITIALIZER_PATH = path.join(getPaths().api.config, 'dbInstance')
```

Comment:
maybe `api/src/prisma/dbInstance.js|ts` is a good place for this?

Reply:
Is it weird to have both api/prisma and api/src/prisma?

Reply:
Maybe `api/src/config` or `api/src/init` or `api/src/initializers`?

Reply:
Doh, I thought it was in `src/prisma` for some reason. I'm trying to think of other instances that we might want to init?

Reply:
Don't know of anything at the moment, but if we move it to somewhere generic then we can use it for all the new stuff we can't foresee yet! I know this one is extremely prisma-specific but the next thing probably won't be...

Reply:
Yarbo, that's true. I like `api/src/init`

Reply:
@cannikin don't hate me, but I actually think `api/src/config` is the most appropriate place... since this is only used when you want to configure you database instance.

Reply:
😐

Reply:
haha Update coming

Reply:
Okay let me know how that looks!